Worktodo.cpp

```diff
@@ -98,12 +102,12 @@ std::optional<Task> Worktodo::getTask(Args &args) {
  again:
   // Try to get a task from the local worktodo.txt
   if (optional<Task> task = firstGoodTask(worktodoTxt)) {
-    if (task->kind == Task::PRP && task->wantsPm1) {
+    if ((task->kind == Task::LL and !task->wantsPm1) or (task->kind == Task::PRP and task->wantsPm1)) {
       // Some worktodo tasks can be expanded into subtasks:
       // PRP with wantsPm1>0 is expanded into a sequence of P-1 followed by PRP with wantsPm1==0.
       Task pm1{Task::PM1, task->exponent, task->AID, "", task->B1, task->B2, task->bitLo, task->wantsPm1};
       pm1.adjustBounds(args);
-      task->wantsPm1 = 0;
+      task->wantsPm1 = task->kind == Task::LL ? 1 : 0;
```

Comment:
I would paranthesize the condition. Maybe even the whole RHS expression.

Reply:
Do you mean you want this: `task->wantsPm1 = (task->kind == Task::LL ? 1 : 0);`?

This line could of course be simplified to just: `task->wantsPm1 = task->kind == Task::LL;`, but I thought that might be confusing.

Reply:
No, I meant
wantsPm1 = (kind == LL) ? 1 : 0;
or even better:
wantsPm1 = (kind == LL);
Anyway not a big deal.